### PR TITLE
login: add page

### PR DESCRIPTION
**Pull request by en1gma713**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by en1gma713** [comment]
#2214

File: pages/linux/login.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Login
+
+> Initiates a session for a user.
+
+- Login as user:
+
+`login [user]`
```
Comment (owenvoke) [inline pages/linux/login.md:7]:
```suggestion
- Login as a user:

`login {{user}}`
```

File: pages/linux/login.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Login as user without authentication if user is preauthenticated:
+
+`login -f [user]`
```
Comment (owenvoke) [inline pages/linux/login.md:11]:
```suggestion
- Login as a user without authentication if user is preauthenticated:

`login -f {{user}}`
```

File: pages/linux/login.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Login as user and preserve environment:
+
+`login -p [user]`
```
Comment (owenvoke) [inline pages/linux/login.md:15]:
```suggestion
- Login as a user and preserve the environment:

`login -p {{user}}`
```

File: pages/linux/login.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+- Login as user with remote host:
+
+`login -h [host] [user]`
```
Comment (owenvoke) [inline pages/linux/login.md:19]:
```suggestion
- Login as a user on a remote host:

`login -h {{host}} {{user}}`
```

File: pages/linux/login.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Login
```
Comment (owenvoke) [inline pages/linux/login.md:1]:
```suggestion
# login
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
Hi @en1gma713, just a few small grammatical changes and token syntax additions, but otherwise this looks good. Thanks!

**Comment by en1gma713** [comment]
done

File: pages/linux/login.md (hunk 5 above)
Comment (owenvoke) [reply pages/linux/login.md:1]: @en1gma713, just this one and should be fine. 👍🏻 We tend to use lowercase titles for the pages. Cheers. 🙂

**Review by owenvoke — APPROVED** [review]
LGTM! Thanks @en1gma713. 🥳

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**